src/components/NewQuestionDialog.js

```diff
@@ -29,9 +31,27 @@ const NewQuestionDialog = ({ open, handleClose, onSubmit }) => {
       type: qType,
     };
 
+    if (values.question == "") {
```

Comment:
Should this be '===' instead?

Reply:
Yah it should be. Fixed.

Reply:
They're functionally the same in this context